src/opm/output/eclipse/WriteRFT.cpp

```diff
@@ -650,14 +657,15 @@ namespace {
                 return;
             }
 
-            this->addConnection(usys, well, connPos, *xconPos.value());
+            const double cell_depth = grid.getCellDepth(connPos->global_index());
+            this->addConnection(usys, well, cell_depth, connPos, *xconPos.value());
         });
     }
 
     void PLTRecord::write(::Opm::EclIO::OutputStream::RFT& rftFile) const
     {
-        rftFile.write("CONDEPTH", this->depth_);
-        rftFile.write("CONPRES" , this->pressure_);
+        rftFile.write("CONDEPTH", this->conn_depth_);
+        rftFile.write("CONPRES" , this->conn_pressure_);
```

Comment:
Using this definition you can just rename the `depth_` and `pressure_` arrays.  There's no need to add the new `conn_*` versions of the arrays since the original ones won't be used anymore.